lab-saul/lib/server.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+// node modules (none needed in this app)
+// npm modules
+const express = require('express');
+const mongoose = require('mongoose');
+// app modules
+// module logic
+// configure mongoose
+mongoose.Promise = Promise;
+mongoose.connect(process.env.MONGODB_URI);
+
+let server;
+const app = express();
+// load middleware
+// load routes
+app.get('/api/hello', (req, res, next) => {
```

Comment:
It's important to keep clutter down in your codebase by removing lines like this one that we're not using anymore.